### ISSUE TITLE
Deployment Error

### ISSUE THREAD

**Issue by ronitkothari** [mansi2024/Yelpcamp#3]
If I click the link [https://dry-ridge-06863.herokuapp.com/](https://dry-ridge-06863.herokuapp.com/), to see the project live, I get an "Application error" from Heroku

**Comment by mansi2024** [comment]
Yes as the heroku became paid for deployment that's why it stopped working. As I have deployed it in heroku.
